py/server/tests/test_table.py

```diff
@@ -902,6 +903,42 @@ def make_pairs_3(tid, a, b):
             self.assertEqual(x2.size, 10)
             self.assertEqual(x3.size, 10)
 
+    def test_callable_attrs_in_query(self):
+        input_cols = [
+            datetime_col(name="DTCol", data=[dtypes.DateTime(1), dtypes.DateTime(10000000)]),
+        ]
+        test_table = new_table(cols=input_cols)
+        from deephaven.time import year, TimeZone
+        rt = test_table.update("Year = (int)year(DTCol, TimeZone.NY)")
+        self.assertEqual(rt.size, test_table.size)
+
+        class Foo:
```

Comment:
The java code above is working with Python attributes.  From a little googling, python attributes can be instance methods, static methods, and class methods.  We are only testing instance methods.  Tests should be included for static methods and class methods to make sure they also work.

Reply:
This tests an instance attribute, but it's not testing a method. These tests cover class, instance, and static methods *and* variables if the PR aims to make all of them usable from query strings.